packages/synapse-react-client/src/lib/containers/auth/LoginPage.tsx

```diff
@@ -1,62 +1,170 @@
 import * as React from 'react'
-import Login from './Login'
-import IconSvg from '../IconSvg'
-import { IconButton } from '@mui/material'
+import {
+  Box,
+  Stack,
+  styled,
+  Typography,
+  TypographyProps,
+  useTheme,
+} from '@mui/material'
+import { LeftRightPanel } from '../../components/styled/LeftRightPanel'
+import useShowDesktop from '../../utils/hooks/useShowDesktop'
+import LoginFlowBackButton from './LoginFlowBackButton'
+import useLogin from '../../utils/hooks/useLogin'
+import LoginForm from './LoginForm'
+import { StyledComponent } from '@emotion/styled'
 
 export type LoginPageProps = {
   ssoRedirectUrl?: string
   sessionCallback: () => void // Callback is invoked after login
 }
 
-function LoginPage(props: LoginPageProps) {
+const Tagline: StyledComponent<TypographyProps> = styled(Typography, {
+  label: 'Tagline',
+})(({ theme }) => ({
+  marginBottom: theme.spacing(2),
+  font: '300 24px/34px Lato, sans-serif',
+}))
+
+export default function LoginPage(props: LoginPageProps) {
```

Comment:
Could consider making logo, text swappable with props/theme/other context to reuse in SageAccountWeb

Reply:
With the current changes, should SageAccountWeb be changed to use StandAloneLoginForm?